integrated_channels/cornerstone/utils.py

```diff
@@ -3,9 +3,28 @@
 Utilities for Cornerstone integrated channels.
 """
 
+import re
 from logging import getLogger
+from uuid import uuid4
+
+from django.apps import apps
+
+from integrated_channels.utils import encode_course_key_into_base64
+
+
+def cornerstone_learner_data_transmission_audit():
+    """
+    Returns the ``EnterpriseCustomer`` class.
+    """
+    return apps.get_model('cornerstone', 'CornerstoneLearnerDataTransmissionAudit')
+
+
+def cornerstone_course_key():
```

Comment:
I would suggest using a `_model` suffix to the function name for better usage clarity